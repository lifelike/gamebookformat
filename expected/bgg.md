# Title of Game #
Your Name

## Introduction ##
 For information that goes before the actual numbered sections, if needed. 

## More Introduction ##
  Any number of these optional pre-game sections are possible really. It would be useful to be able to do sub-sections here to mark up a complete rulebook as part of the paragraph booklet, but there is no such feature (yet?). 

Turn to 1 to begin.
<a name="section1">

## 1 ##
 This is the first section (or paragraph, if you prefer). From here you can go to section [4](#section4)
 or [3](#section3)
. 
<a name="section2">

## 2 ##
 This is the end. There is nothing more in this gamebook.
<a name="section3">

## 3 ##
 Third section. Again, its number is not known. You can go to the end at [2](#section2)
. This section has an image to show the syntax for doing that (same as images in bgg forums really). ![](testimage.png)
 
<a name="section4">

## 4 ##
 This is the second section. It may or may not end up with number 2 in the generated gamebook. From here you can skip to the [2](#section2)
 or go to the third section at [3](#section3)
. 

