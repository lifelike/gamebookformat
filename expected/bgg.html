<!DOCTYPE html>
<html>
  <head>
    <meta charset='utf-8'>
    <meta name="viewport"
        content="width=device-width, initial-scale=1, maximum-scale=1">
    <title>Title of Game</title>
    <script type="text/javascript" src="gamebookformatplay.js"></script>
    <link rel="stylesheet" href="gamebookformat.css"
          type="text/css" />
    <script>
      if (typeof gamebook !== 'undefined') {
         gamebook.id = 'bgg';
      }
    </script>
  </head>
  <body>
<div class="hideintrolink nodisplay"
    onclick="gamebook.hideIntroSections()">(hide instructions)</div>
  <div class="gamebook">
<div class="introsection">
<div class="introsectionheading">Introduction</div>
<div class="introsectionbody">
 For information that goes before the actual numbered sections, if needed. 
</div>
</div>
<div class="introsection">
<div class="introsectionheading">More Introduction</div>
<div class="introsectionbody">
  Any number of these optional pre-game sections are possible really. It would be useful to be able to do sub-sections here to mark up a complete rulebook as part of the paragraph booklet, but there is no such feature (yet?). 
</div>
</div>
<div class="resumelink nodisplay"
  onclick="gamebook.loadGame()">Resume saved game.</div>

<div class="startlink"
  onclick="gamebook.turnTo(1)">Turn to 1 to begin.</div>
<script>
    if (typeof gamebook !== 'undefined' && gamebook.hasSavedGame()) {
        var resumeLinks = document.getElementsByClassName('resumelink');
        Array.prototype.forEach.call(resumeLinks, function(e) {
            e.classList.remove('nodisplay');
        });
    }
</script>
<div class="section" id="section1"
     >
<div class="sectionnumber" id="para1">1</div>
<div class="sectiontext">
 This is the first section (or paragraph, if you prefer). From here you can go to section <a class="sectionref enabledlink" data-ref="4"
href="#section4">4</a> or <a class="sectionref enabledlink" data-ref="3"
href="#section3">3</a>. 
</div>
</div>
<script>
if (typeof gamebook !== 'undefined') {
   gamebook.addSection(1, document.getElementById('section1'));
}
</script>
<div class="section" id="section2"
     >
<div class="sectionnumber" id="para2">2</div>
<div class="sectiontext">
 This is the end. There is nothing more in this gamebook.
</div>
</div>
<script>
if (typeof gamebook !== 'undefined') {
   gamebook.addSection(2, document.getElementById('section2'));
}
</script>
<div class="section" id="section3"
     >
<div class="sectionnumber" id="para3">3</div>
<div class="sectiontext">
 Third section. Again, its number is not known. You can go to the end at <a class="sectionref enabledlink" data-ref="2"
href="#section2">2</a>. This section has an image to show the syntax for doing that (same as images in bgg forums really). <img src="testimage.png" class="sectionimage"></img> 
</div>
</div>
<script>
if (typeof gamebook !== 'undefined') {
   gamebook.addSection(3, document.getElementById('section3'));
}
</script>
<div class="section" id="section4"
     >
<div class="sectionnumber" id="para4">4</div>
<div class="sectiontext">
 This is the second section. It may or may not end up with number 2 in the generated gamebook. From here you can skip to the <a class="sectionref enabledlink" data-ref="2"
href="#section2">2</a> or go to the third section at <a class="sectionref enabledlink" data-ref="3"
href="#section3">3</a>. 
</div>
</div>
<script>
if (typeof gamebook !== 'undefined') {
   gamebook.addSection(4, document.getElementById('section4'));
}
</script>
    <div id="counters" class="counters">
    </div>
    <div id="counterTemplate" class="counterTemplate">
      <span class="counterheading"></span>
      <span class="countercontents"></span>
    </div>
    </div>
    <div id="collections" class="collections">
    </div>
    <div id="collectionTemplate" class="collectionTemplate">
      <span class="collectionheading"></span>
      <span class="collectioncontents"></span>
    </div>
    </div>
    <script>
if (typeof gamebook !== 'undefined') {
   gamebook.prepare();
}
    </script>
<div class="displayintrolink nodisplay"
    onclick="gamebook.showIntroSections()">(show instructions)</div>
    </div>
  </body>
</html>
